src/App.tsx

```diff
@@ -20,17 +25,125 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    itemWidth: 130,
+    step: 3,
+    frameSize: 3,
+    animationDuration: 1000,
+    infinite: false,
+  };
+
+  changeItemWidth = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (+event.target.value >= 130) {
```

Comment:
```suggestion
    const width = +event.target.value;

    if (width >= 130) {
```

If you are doing something >= 2 times better create a variable to hold processed value.